Ginger/Ginger/ApplicationModelsLib/POMModels/POMWizardLib/LearnWizard/POMGeneralDetailsWizardPage.xaml.cs

```diff
@@ -113,5 +120,36 @@ private void xBrowseImageLoadButton_Click(object sender, RoutedEventArgs e)
                 }
             }
         }
+
+        private void XBusinessFlowControl_ElementChangedPageEvent()
+        {            
```

Comment:
converting selected BF to RepositoryItemKey and adding it to POM should be done inside the ucBusinssFlow user control and not on use page- because then you will need to repeat this code on all pages.
the ucBusinssFlowMap should get as input the parent object (in this case POM) and the name of the field to set the BF key on- and then the ucBusinssFlowMap should load from it previously selected BF and store on it selected BF- same like we did on ucPOMSelector

Reply:
repositorykey code added to ucBusinessFlow user control

Reply:
I meant that all BF mapping set/load will be done inside the user control like we do for POM- if not clear i will explain in meeting